Modules/Projects/Core/Script_TableEnums.lua

```diff
@@ -37,5 +37,11 @@ ECharacterFlags = {
     SkipTurns       = 16,
 }
 
+EVoreStyle = {
+    OV              =0,
```

Comment:
Pedantic: please put spaces before _and_ after the `=`, same as the other enums in this file.